packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -279,7 +283,32 @@ export default class AwsS3Multipart extends BasePlugin {
     })
   }
 
-  uploadRemote (file) {
+  #requestSocketToken = async (file) => {
+    const Client = file.remote.providerOptions.provider ? Provider : RequestClient
+    const client = new Client(this.uppy, file.remote.providerOptions)
+    const opts = { ...this.opts }
+
+    if (file.tus) {
+      // Install file-specific upload overrides.
+      Object.assign(opts, file.tus)
+    }
+
+    try {
+      // !! cancellation is NOT supported at this stage yet
```

Comment:
That seems like a blocker, right?

Reply:
This has always been the case and we can't do anything about it AFAIK

Reply:
Not sure what you mean by that, cancellation seems to be supported on the `main` branch (there are no more requests sent once the user presses the cancel button)

Reply:
I mean this comment has always been there and we can't cancel something at this stage because there is no socket connection and you need a socket connection to cancel something.